src/filesystem/index.ts

```diff
@@ -34,26 +45,117 @@ function expandHome(filepath: string): string {
   return filepath;
 }
 
-// Store allowed directories in normalized form
-const allowedDirectories = args.map(dir =>
-  normalizePath(path.resolve(expandHome(dir)))
-);
+// Default exclusion patterns that always apply
+const DEFAULT_EXCLUSIONS = ['.git', 'node_modules'];
```

Comment:
```suggestion
const DEFAULT_EXCLUSIONS = [
'.git',
'node_modules',
'.env*',
'id_rsa',
'id_dsa',
'*.pem'
];
```
(So this could also exclude common secrets/key files by default)